mapbox/src/main/java/org/odk/collect/mapbox/MapboxMapFragment.kt

```diff
@@ -95,7 +95,7 @@ class MapboxMapFragment :
     private var tileServer: TileHttpServer? = null
     private var referenceLayerFile: File? = null
     private var clientWantsLocationUpdates = false
-    private var offlineLayerPosition = -1
+    private var topStyleLayerId = ""
```

Comment:
It could be but I can't see any benefit in this case. Could you explain why it's better to you? It's always set before we use it. Even if it's not set we do not block adding offline layers.

Reply:
Ah will it not matter if we pass `""` to `addLayerAbove`?

Reply:
It shouldn't I guess but ok let's not do that.